src/main/resources/application.properties

```diff
@@ -153,3 +153,6 @@ tls.pulsar.admin.ca-certs=ca-client-path
 
 # support peek message, default false
 pulsar.peek.message=false
+
+# swagger configration
+swagger.enabled=true
```

Comment:
I suggest we disabled it by default, many users will deploy it directly online, which may bring some security issues, if users want to enable it on, they can configure it themselves

Reply:
yes, good suggestion, I set this property false, and commit again